docker/docker-compose.dev.yml

```diff
@@ -7,5 +7,10 @@ services:
             dockerfile: ./docker/Dockerfile
         image: aegee/frontend:dev
         volumes:
-            - ./${PATH_FRONTEND}/../:/usr/app/src
-            - /usr/app/src/node_modules
```

Comment:
are you sure about removing this?

Reply:
yep, it's the same way it works for all other services.